packages/launcher/src/renderer/ModalView.js

```diff
@@ -0,0 +1,40 @@
+//@flow
+
+import React, {
+  createRef,
+  Component,
+  type Element,
+  type ElementRef,
+} from 'react'
+import { View, StyleSheet, Text } from 'react-native-web'
+import ReactModal from 'react-modal'
+
+type Props = {
+  onRequestClose: () => void,
+  children?: ?Element<any>,
```

Comment:
Better not use the `any` type when possible as it's essentially untyped, for `children` if you just want to accept anything provided, the `Node` type is more relevant, see https://flow.org/en/docs/react/types/#toc-react-node